pkg/cmd/admin/prune/images.go

```diff
@@ -99,6 +115,16 @@ func (o *PruneImagesOptions) Complete(f *clientcmd.Factory, cmd *cobra.Command,
 		return kcmdutil.UsageError(cmd, "no arguments are allowed to this command")
 	}
 
+	if !cmd.Flags().Lookup("keep-younger-than").Changed {
```

Comment:
I'll check, I thought it's specifically when flag is set by a user, no matter which value.